package.json

```
@@ -5,7 +5,7 @@
         "dev": "dotenv -c secrets -- dev-pm start",
         "copy-schema-files": "node copy-schema-files.js",
         "lint": "run-p lint:*",
-        "lint:root": "npx prettier --check './**/*.{js,json,md,yml,yaml}'",
+        "lint:root": "npx prettier --check './!(admin|api|site|create-app)/**/*.{js,json,md,yml,yaml}'",
```

now that this doesn't lint root alone anymore, maybe rename to:
```suggestion
        "lint:prettier": "npx prettier --check './!(admin|api|site|create-app)/**/*.{js,json,md,yml,yaml}'",
```

---

or, wait: des the `!` mean those will be excluded?

---

ok, so why this change? does it still fix "bad IDE support"?